server/bin/timely-standalone.sh

```diff
@@ -25,12 +25,15 @@ if [[ -e ${NATIVE_DIR} ]]; then
 fi
 mkdir -p ${NATIVE_DIR}
 
-$JAVA_HOME/bin/jar xf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so
+pushd ${BASE_DIR}/bin
+echo "$JAVA_HOME/bin/jar xvf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so"
+$JAVA_HOME/bin/jar xvf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so
```

Comment:
I don't know what the best answer is here. I guess we could do some bash to check the os and switch to whatever the right file extension is. However, if you just use the docker from a Mac, it's a non-issue.